convert/src/label_studio_to_badgerdoc/models/label_studio_models.py

```diff
@@ -95,21 +105,21 @@ class ModelItem(BaseModel):
     annotations: List[Annotation]
     file_upload: str = ""
     drafts: List = []
-    predictions: List[int]
+    predictions: List[int] = []
     data: Data
     meta: Dict[str, Any] = {}
-    created_at: str = "2022-12-13T09:57:08.451845Z"
-    updated_at: str = "2022-12-13T09:57:08.451845Z"
+    created_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
+    updated_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
     inner_id: int = 1
     total_annotations: int = 0
     cancelled_annotations: int = 0
     total_predictions: int = 0
-    comment_count: int = 0
-    unresolved_comment_count: int = 0
-    last_comment_updated_at: Any = ""
+    comment_count: Optional[int] = 0
+    unresolved_comment_count: Optional[int] = 0
+    last_comment_updated_at: Optional[Any] = ""
```

Comment:
May you clarify this type? It can be string and data? A bit strange behaviour

Reply:
Added None as default

Reply:
But what the actual type? I mean Any in this case. Maybe it should be datetime or str?

Reply:
https://github.com/epam/badgerdoc/issues/170